.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "julia.environmentPath": "/home/melonedo/LibGEOS.jl/gen"
+}
```

Comment:
Could you remove this file? You could also add `.vscode` to `.gitignore` in this repo.